### DISCUSSION TITLE
DSWx-HLS Engineering Release 2.0 Discussion

### DISCUSSION THREAD

**Discussion by sdcolli1** [nasa/opera-sds-pge#58] [Show and tell]
Discussion page for Engineering Release 2.0 for the DSWx-HLS PGE.

Feel free to use this space to field any questions about the release, current capabilities, planned capabilities for ER3.0, etc...

**Comment by riverma** [comment]
Thanks for setting up a thread for your release @collinss-jpl 👍 . Great idea.
